package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "build": "webpack --config webpack.config.js",
     "build-watch": "webpack --watch --config webpack.config.js",
-    "dev-server": "webpack-dev-server --config webpack.config.js"
+    "dev-server": "webpack-dev-server --config webpack.config.js",
+    "test": "./node_modules/.bin/tslint -p ."
```

Comment:
@AnshulMalik  Please change this to `"tslint": ""./node_modules/.bin/tslint -p ."` as unit and integration test scripts will come here,otherwise conflicts will be there.